roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -125,10 +124,15 @@
     - ansible_distribution == "CentOS"
     - ansible_distribution_major_version == "7"
 
-
 - name: Write cri-o proxy drop-in
   template:
     src: http-proxy.conf.j2
     dest: /etc/systemd/system/crio.service.d/http-proxy.conf
   notify: restart crio
   when: http_proxy is defined or https_proxy is defined
+
+- name: Ensure crio service is started and enabled
+  service:
+    name: crio
+    enabled: yes
+    state: started
```

Comment:
why do you think it is needed here again? there is already a handler for

Reply:
Because I had a special case with crio installed but stopped. The role does not handle that case. The docker role has a similar task

Reply:
true, if nothing changes but crio is stopped